### DISCUSSION TITLE
Using Themed Tool Windows

### DISCUSSION THREAD

**Discussion by sharpninja** [VsixCommunity/Community.VisualStudio.Toolkit#484] [General]
I have a question about theming with the Community Toolkit. WPF elements such as `ComboBox`, `TextBox` and `Button` have really large fonts, padding and margins when using `toolkit:Themes.UseVsTheme="True"`. How can I control this in a single location?

**Comment by RobertvanderHulst** [comment]
You should add a ThemeDictionary to your project. 
See https://github.com/X-Sharp/XSharpPublic/blob/main/src/VisualStudio/Debugger.UI/Controls/VsThemeDictionary.xaml
The ThemeDictionary can retrieve colors from other sources. Ours gets most of its colors from Microsoft.VisualStudio.Shell.15.0.
Then in the xaml (when needed) tell VS that you want to use this dictionary:
See for example
https://github.com/X-Sharp/XSharpPublic/blob/main/src/VisualStudio/Debugger.UI/Controls/GlobalsControl.xaml#L33
